earthpy/spatial.py

```diff
@@ -44,20 +44,20 @@ def extent_to_json(ext_obj):
 # Calculate normalized difference between two arrays
 # Both arrays must be of the same size
 
-def normalized_diff(b1, b2):
-    """Take two numpy arrays and calculate the normalized difference
+def normalized_diff(b2, b1):
```

Comment:
This is great @jlpalomino . i think that perhaps my comment was unclear. 
did we decide to do something like
```
def normalized_diff(b1, b2):

Math will be calculated (b1-b2) / (b1+b2)

...
b1, b2 : arrays with the same shape

ndiff = es.normalized_diff(b1=nir_band, b2=red_band)
```

@joemcglinchy does this follow what you and i think jenny had +1 in the issue ??

What is the most intuitive to do here for a user. for me i'm wondering if we just want b1, b2 and then math is calculated b1-b2. but perhaps i'm odd in my thinking!

Reply:
@lwasser The band order has been updated to match your example of b1 and b2, per the group's decision.